### ISSUE TITLE
git fork keep loading all repos

### ISSUE THREAD

**Issue by Xabier456** [ForkIssues/TrackerWin#1994]
I installed fork on a new laptop and when I want to clone any repository no matter what it is it keeps on keep loading or doesn't show the commits or branches.
for some reason this problem does not happen with my other pc.

🔷 2023-08-27 02:03:04.437 WindowActivated
🔷 2023-08-27 02:03:04.437 Application Window Activated
🔷 2023-08-27 02:03:04.437 RefreshRepositoryData
◽️ 2023-08-27 02:03:04.437 proyectoExpressLaboComp4 RefreshRepositoryDataGitCommand All
◽️ 2023-08-27 02:03:05.920 Saving settings.
◽️ 2023-08-27 02:03:05.974 Saving settings.
🔷 2023-08-27 02:03:15.673 WindowActivated
🔷 2023-08-27 02:03:15.673 Application Window Activated
◽️ 2023-08-27 02:03:18.622     242ms: GET https://api.github.com/user (200)
⚠️ 2023-08-27 02:03:18.622 Cannot parse 'name'
◽️ 2023-08-27 02:03:20.262     371ms: GET https://api.github.com/user/repos?page=1&per_page=70 (200)
◽️ 2023-08-27 02:03:31.240 Cloning into 'C:\Users\Xabier\Desktop\UTN TUP\repos git\proyectoExpressLaboComp4'...
◽️ 2023-08-27 02:03:31.823 Fork_Pipe2784_AskPass: received event '362'
◽️ 2023-08-27 02:03:31.823 Fork_Pipe2784_AskPass: waiting for next event '295'
◽️ 2023-08-27 02:03:33.163 POST git-upload-pack (185 bytes)
◽️ 2023-08-27 02:03:33.420 POST git-upload-pack (212 bytes)
◽️ 2023-08-27 02:03:33.647 remote: Enumerating objects: 3, done.        
◽️ 2023-08-27 02:03:33.647 remote: Counting objects:  33% (1/3)        
◽️ 2023-08-27 02:03:33.647 remote: Counting objects:  66% (2/3)        
◽️ 2023-08-27 02:03:33.647 remote: Counting objects: 100% (3/3)        
◽️ 2023-08-27 02:03:33.647 remote: Counting objects: 100% (3/3), done.        
◽️ 2023-08-27 02:03:33.647 remote: Total 3 (delta 0), reused 0 (delta 0), pack-reused 0        
◽️ 2023-08-27 02:03:33.735 Receiving objects:  33% (1/3)
◽️ 2023-08-27 02:03:33.735 Receiving objects:  66% (2/3)
◽️ 2023-08-27 02:03:33.738 Receiving objects: 100% (3/3)
◽️ 2023-08-27 02:03:33.738 Receiving objects: 100% (3/3), done.
◽️ 2023-08-27 02:03:35.422 Saving settings.
🔷 2023-08-27 02:03:35.422 RefreshRepositoryData
◽️ 2023-08-27 02:03:35.422 proyectoExpressLaboComp4 RefreshRepositoryDataGitCommand All
◽️ 2023-08-27 02:03:35.422 Loading proyectoExpressLaboComp4 settings.
◽️ 2023-08-27 02:03:35.422 Loading proyectoExpressLaboComp4 settings.
🔷 2023-08-27 02:03:35.422 Use default repository settings
🔷 2023-08-27 02:03:35.422 Use default repository settings
◽️ 2023-08-27 02:03:35.476 Saving settings.
🔷 2023-08-27 02:03:39.750 WindowActivated
🔷 2023-08-27 02:03:39.750 Application Window Activated: skip (CloneWindow)

**Comment by DanPristupov** [comment]
Do you have .net 4.6.2 installed?

In the other topic, it was mentioned that installing Windows SDK helped. I suppose because it installed .net. https://github.com/fork-dev/TrackerWin/issues/1990#issuecomment-1694155457

**Comment by Xabier456** [comment]
I installed the windows SDK and .net, restarted the pc and reinstalled fork and it is still loading the repos :/

**Comment by DanPristupov** [comment]
I think I found the problem. Because we build Fork with the latest Visual Studio build tools, it requires 'Microsoft Visual C++ Redistributable' package.

https://aka.ms/vs/17/release/vc_redist.x64.exe
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

I just reproduced the problem on a clean virtual machine and solved it by installing that package.
